editor.planx.uk/src/pages/PlatformAdminPanel.tsx

```diff
@@ -0,0 +1,167 @@
+import Close from "@mui/icons-material/Close";
+import Done from "@mui/icons-material/Done";
+import Accordion from "@mui/material/Accordion";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import AccordionSummary from "@mui/material/AccordionSummary";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { SummaryListTable } from "@planx/components/shared/Preview/SummaryList";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+import { AdminPanelData } from "types";
+import Caret from "ui/icons/Caret";
+
+const StyledTeamAccordion = styled(Accordion, {
+  shouldForwardProp: (prop) => prop !== "primaryColour",
+})<{ primaryColour?: string }>(({ theme, primaryColour }) => ({
+  borderLeft: `10px solid ${primaryColour}`,
+  backgroundColor: theme.palette.background.paper,
+  width: "100%",
+  position: "relative",
+  marginBottom: theme.spacing(2),
+  padding: theme.spacing(1),
+  "&::after": {
+    position: "absolute",
+    width: "100%",
+  },
+}));
+
+function Component() {
+  const adminPanelData = useStore((state) => state.adminPanelData);
+
+  return (
+    <Box p={3}>
+      <Typography variant="h1">Platform Admin Panel</Typography>
+      <Typography variant="body1" mb={3}>
+        {`This is an overview of each team's integrations and settings for the `}
+        <strong>{process.env.NODE_ENV}</strong>
```

Comment:
Question: any ideas of a better/more accurate way to display the environment here? We have a nice `getFormattedEnvironment()` helper in the API, but I'm looking for an alternative using frontend/React env vars only?

Eg pizza is currently displaying "production"
![Screenshot from 2024-04-15 10-07-43](https://github.com/theopensystemslab/planx-new/assets/5132349/15adb91b-9cfd-4baa-9ded-96bc937cc039)